hathor/transaction/scripts/opcode.py

```diff
@@ -82,7 +82,8 @@ class Opcode(IntEnum):
     OP_DATA_MATCH_VALUE = 0xD1
     OP_SIGHASH_BITMASK = 0xE0
     OP_SIGHASH_RANGE = 0xE1
-    OP_MAX_INPUTS_OUTPUTS = 0xE2
+    OP_MAX_SIGHASH_SUBSETS = 0xE2
+    OP_MAX_INPUTS_OUTPUTS = 0xE3
```

Comment:
Just to double check: It is safe to change the OPCODE because these opcodes haven't been activated in any network yet. Is that correct?